webapp/src/components/ProgramCalendar.tsx

```diff
@@ -65,6 +75,54 @@ const ProgramCalendar = ({
   const [dialogContents, setDialogContents] = useState<CalendarEvent>(
     defaultDialogContents
   );
+  const [activitiesCompletion, setActivitiesCompletion] =
+    React.useState<ParticipantActivityForProgram>({
```

Comment:
One more React I didn't delete. Please delete it

Reply:
Good catch, thanks!